layers/restclient/README.org

```diff
@@ -22,6 +22,7 @@ file.
 
 | Key Binding | Description                                                 |
 |-------------+-------------------------------------------------------------|
+| ~SPC m c~   | Copy query under the cursor as a curl command               |
```

Comment:
I have never used this package so I might be wrong, but  I think `y` (which stands for `yank`) instead of `c` would
 fit better Spacemacs' conventions